### DISCUSSION TITLE
Access another server on the internal network via HA et cloudflare

### DISCUSSION THREAD

**Discussion by gypais** [homeassistant-apps/app-cloudflared#689] [Q&A]
My cloudflared configuration works perfectly. So I access HA local remotely. Now I want to access my frigate server on my network internally at home. Where can I find the documentation to make proxypath so that the URL /my-frigate of my HA server redirects to a server in 192.168.0.34 for example?
PS: maybe it has nothing to do with cloudflare!
Many thanks.

**Comment by brenner-tobias** [comment]
Hi. Unfortunately, redirecting specifics paths is not possible using only Cloudflared. I raised an [issue for that](https://github.com/cloudflare/cloudflared/issues/563) quite some time ago, but it was not implemented yet.
What you can do is use another subdomain for what you want to do (e.g. my-frigate.example.com), which can be easily archived by using the option [additional hosts](https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#option-additional_hosts).
Let me know if there is anything else that you need help with.
